packages/relay/src/lib/errors/JsonRpcError.ts

```diff
@@ -111,6 +111,11 @@ export const predefined = {
     code: 32001,
     message: `Nonce too low. Provided nonce: ${nonce}, current nonce: ${currentNonce}`
   }),
+  'NONCE_TOO_HIGH': (nonce, currentNonce) => new JsonRpcError({
```

Comment:
Could also repurpose `INCORRECT_NONCE` error in this file if desired. 
Optionally, we can leave as is and later just remove `INCORRECT_NONCE` as it's not used.

Reply:
so instead of using `NONCE_TOO_HIGH` or `NONCE_TOO_LOW`, we just use the `INCORRECT_NONCE` right?

Reply:
I think that remove `INCORRECT_NONCE` would be better, so the errors are more explicitly how is right now.
✅ Done